### PR TITLE
Iz gawb-3169 import tutorial

### PR DESCRIPTION
**Pull request by zarsky-broad**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [x] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [x] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [x] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [x] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: src/cljs/main/broadfcui/page/external_importer.cljs [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+      (links/create-external
+       {:style {:display "block" :marginTop "0.3rem"}
+        :href (config/billing-account-guide-url)}
+       "Click here to for instructions.")]]
+    [:div {:style {:display "inline-block"}}
+     [:div {:style {:display "inline-block"
+                    :background-color (:button-primary style/colors) :color "white"
+                    :fontSize "1.25rem" :textAlign "center" :lineHeight "2rem"
+                    :height "2rem" :width "2rem" :borderRadius "50%"}}
+      "2"]
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/external_importer.cljs]: Extract the creation of this div into a function

File: src/cljs/main/broadfcui/page/external_importer.cljs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+   "In order to import data into FireCloud, you'll need two things:"
+   [:div {:style {:marginTop "1.5rem"}}
+    [:div {:style {:width "50%" :display "inline-block"}}
+     [:div {:style {:display "inline-block"
+                    :background-color (:button-primary style/colors) :color "white"
+                    :fontSize "1.25rem" :textAlign "center" :lineHeight "2rem"
+                    :height "2rem" :width "2rem" :borderRadius "50%"}}
+      "1"]
+     [:div {:style {:display "inline-block" :verticalAlign "top" :marginLeft "0.5rem"}}
+      [:h2 {:style {:margin "0 0 0.3rem"}} "Google Billing Project"]
```
Comment (MattPutnam) [inline src/cljs/main/broadfcui/page/external_importer.cljs]: Do we need the explicit top/side margins of 0 to override h2 defaults?

**Review by MattPutnam — APPROVED** [review]
👍 One small thing and one question

File: src/cljs/main/broadfcui/page/external_importer.cljs (hunk 2 above)
Comment (zarsky-broad) [reply src/cljs/main/broadfcui/page/external_importer.cljs]: Just the top; the sides are there so that I don't have separate props. Would you prefer that?

**Comment by davidangb** [comment]
@zarsky-broad  I removed myself as reviewer - as long as you get at least two thumbs, I'm good with this. If the PR is still open next week, I can review.

**Review by rushtong — APPROVED** [review]
👍  LGTM, tested locally!

**Review by davidangb — CHANGES_REQUESTED** [review]
:+1: for code syntax.

I have problems with the content, which I realize is probably not your doing:

* the title "Import Docker Image to FireCloud" is incorrect. We're importing WDL, not a docker image.
* the subtitle "In order to import data ... " is wrong; we're importing WDL, not data.
* it's possible for a user to successfully import and run WDL without a billing account - as long as that user has write/compute access on a workspace. So the body of the tutorial is incorrect too.

**Review by aednichols — APPROVED** [review]

**Review by davidangb — APPROVED** [review]
:+1:

optional: I find it slightly unbalanced that there's an `external-importer` that defines a title, subtitle, etc - but the "normal" logged out page has its title, subtitle, etc defined inline. I absolutely would not hold the PR to change that, though.

**Pull request merged**